docs/dev/running/aziot-certd.md

```diff
@@ -1,7 +1,10 @@
 # Configuring and running `aziot-certd`
 
-A basic configuration looks like:
+Configuration consists of the main config file (default `/etc/aziot/certd/config.toml`) and any number of principal files in the config directory (`/etc/aziot/certd/config.d`).
 
+The main config file and all files in the config directory must be readable by the user you will run the service as. The default main config file and config directory can be overridden with the environment variables `AZIOT_CERTD_CONFIG` and `AZIOT_CERTD_CONFIG_DIR`, respectively.
+
+Example main config file:
 ```toml
```

Comment:
Add an empty line before the code block.

There are bunch of places in this and the other files.

Reply:
Fixed